### PR TITLE
Main

### PR DESCRIPTION
**Pull request by huytruongkd9**
rưewewew

### CODE REVIEW

**Comment by huytruongkd9** [comment]
êre

**Comment by huytruongkd9** [comment]
dsd

File: UserManagementWithIdentity/Controllers/Api/UsersController.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+using System;
+using System.Threading.Tasks;
+using UserManagementWithIdentity.Models;
+
+namespace UserManagementWithIdentity.Controllers.Api
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    [Authorize(Roles = "Admin")]
+    public class UsersController : ControllerBase
```
Comment (muhammadelhelaly) [inline UserManagementWithIdentity/Controllers/Api/UsersController.cs:13]: First of all thank for your contribution :)
Please add the .vs folder to the .ignore file
regarding this controller please move the delete action to the normal Controller, as in .Net Core no need for api controller to handle this point

File: UserManagementWithIdentity/Controllers/UsersController.cs [hunk 2]
```diff
@@ -37,6 +37,125 @@ public async Task<IActionResult> Index()
             return View(users);
         }
 
+        public async Task<IActionResult> Add()
+        {
+            var roles = await _roleManager.Roles.Select(r => new RoleViewModel { RoleId = r.Id, RoleName = r.Name }).ToListAsync();
```
Comment (muhammadelhelaly) [inline UserManagementWithIdentity/Controllers/UsersController.cs:42]: No need to send the RoleId as you will assign the roles using the role name

File: UserManagementWithIdentity/Views/Users/Add.cshtml [hunk 3]
```diff
@@ -0,0 +1,59 @@
+﻿@model AddUserViewModel
+
+@{
+    ViewData["Title"] = "Add New User";
+}
+
+<form asp-action="Add" method="post">
+    <h4>Create a new user.</h4>
+    <hr />
+    @*<div asp-validation-summary="All" class="text-danger"></div>*@
```
Comment (muhammadelhelaly) [inline UserManagementWithIdentity/Views/Users/Add.cshtml:10]: Remove comment

**Review by muhammadelhelaly — COMMENTED** [review]
Many thanks for your great work :)

**Pull request closed without merging**